jsonnet/kube-thanos/kube-thanos-sidecar.libsonnet

```diff
@@ -0,0 +1,28 @@
+// This example adds the --store flag to the querier to find sidecars.
+// The used values are the defaults in kube-prometheus.
+// If you need to modify this, you most likely want to
+// copy this file into your project and change it accordingly.
+
+{
+  thanos+:: {
+    querier+: {
+      deployment+: {
+        spec+: {
+          template+: {
+            spec+: {
+              containers: [
+                super.containers[0]
+                { args+: [
+                  '--store=dnssrv+_grpc._tcp.%s.%s.svc.cluster.local' % [
+                    'prometheus-k8s',
```

Comment:
should these not be more dynamic?

Reply:
We can't make this more dynamic. This sidecar patch, might be used with kube-prometheus, but probably in different `main.jsonnet` files and even if it would be the same file, I'd still want this to be more generic. As the comment says, think of this more like an example that you copy to your project.